### PR TITLE
Update console.rst

### PR DESCRIPTION
**Pull request by hbgamra**
remove $kernel variable

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
In the Symfony Demo app we recently had to do this change to fix some PHPStan issue: https://github.com/symfony/demo/pull/1457/commits/b9b47c74c5de06900577ef2dd57ba7e5ad9ea400

File: console.rst [hunk 1]
```diff
@@ -514,8 +514,8 @@ console::
     {
         public function testExecute(): void
         {
-            $kernel = self::bootKernel();
-            $application = new Application($kernel);
+            self::bootKernel();
+            $application = new Application(self::$kernel);
```
Comment (theofidry) [inline console.rst:517]: maybe we could go a step further?


```suggestion
            $application = new Application(self::bootKernel());
```

**Review by theofidry — APPROVED** [review]

**Comment by theofidry** [comment]
I am not sure we should worry about PHPStan here however, the variable type is documented correctly and PHPStan does not come with Symfony by default.

**Comment by OskarStark** [comment]
Thank you @hbgamra.

**Pull request merged**